### ISSUE TITLE
After format closure, then compile failed

### ISSUE THREAD

**Issue by keisme** [nicklockwood/SwiftFormat#607]
**Before format:**

// (self, isValid) is a tuple
view.textFieldDidChange.delegate(on: self) { (self, isValid) in 
    // code
}

**After format:**
view.textFieldDidChange.delegate(on: self) { self, isValid in
    // code
}

Thus, compile failed...
How to solve this problem?

**Comment by nicklockwood** [comment]
@keisme thanks for reporting. This looks like a bug - I'll try to get it fixed asap.

There are two possible workarounds, either put backticks around the `self` parameter, i.e.

```swift
view.textFieldDidChange.delegate(on: self) { `self`, isValid in
```

Or disable the redundantParens rule with a comment:

```swift
// swiftformat:disable:next redundantParens
view.textFieldDidChange.delegate(on: self) { (self, isValid) in
```

**Comment by nicklockwood** [comment]
Fixed in 0.44.8
